packages/salesforcedx-vscode-apex-debugger/README.md

```diff
@@ -62,6 +62,13 @@ To start a debugging session, from the configuration dropdown menu at the top of
 
 While a debugging session is in progress, any synchronous activity that runs a line of code with a breakpoint causes execution to halt at the breakpoint. While execution is paused, you can inspect the call stack and see the current values of your variables. You can also step through your code, using the Debug actions pane that appears at the top of the editor while a debugging session is in progress, and watch those values change. You can debug up to two threads at a time. For more information, see [Debugging](https://code.visualstudio.com/docs/editor/debugging) in the Visual Studio Code docs.
 
+## Set Exception Breakpoints
+To make the Apex Debugger halt execution when an exception is thrown during a debugging session, set breakpoints on exceptions. When an exception breakpoint is hit, the debugger pauses on the line of code that caused the exception to be thrown.
```

Comment:
Last sentence - I think you could leave out "to be thrown" and just say "that caused the exception."